app/schemas/com.nextcloud.talk.data.source.local.TalkDatabase/9.json

```diff
@@ -2,7 +2,7 @@
   "formatVersion": 1,
   "database": {
     "version": 9,
-    "identityHash": "666fcc4bbbdf3ff121b8f1ace8fcbcb8",
+    "identityHash": "250a3a56f3943f0d72f7ca0aac08fd1e",
```

Comment:
shouldn't there be a file called 10.json instead to modify 9.json ?

Reply:
Yeah that's weird, I'll just add it in manually I guess. Seems to pass the test.

<img width="469" alt="image" src="https://github.com/nextcloud/talk-android/assets/69230048/940c3cb3-6d81-4bcc-bad5-a8d2b9172a72">